packages/server/lib/cloud/protocol.ts

```diff
@@ -0,0 +1,77 @@
+import fs from 'fs-extra'
+import { NodeVM } from 'vm2'
+import Debug from 'debug'
+import CDP from 'chrome-remote-interface'
+import type { ProtocolManager } from '@packages/types'
+
+interface AppCaptureProtocolInterface {
+  setupProtocol (url?: string): Promise<void>
+  connectToBrowser (options: any): void
+  beforeSpec (spec: any): void
+  afterSpec (): void
+  beforeTest (test: any): void
```

Comment:
Do we want to type any of the `any`s?

Reply:
[`fcb6c6f` (#26080)](https://github.com/cypress-io/cypress/pull/26080/commits/fcb6c6f9833d27a6958eb0a8dbde8449fabc6447#diff-fae7886d3dd070e07a8b39ac5718534511764b7f03bc24c9726f82977b53a225)